external/LibraryThemeStyles/src/Controller/AdminController.php

```
@@ -0,0 +1,184 @@
+
+        $count = 0;
+        foreach ($values as $k => $v) {
+            $current[$k] = $v;
+            $count++;
+        }
+        $siteSettings->set($key, $current);
+
+        return [$count, $current];
+    }
+
+    private function saveSettingsAsPresetDefaults(?string $siteSlug, string $themeKey, string $preset): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$themeKey'. (undefined)

(UnusedFormalParameter)